README.md

```diff
@@ -88,3 +88,22 @@ drop into the shell.
 ### Load addresses:
 
 `./scripts/load-addresses` (from `out/addresses.json` to ENV variables)
+
+## Nix
+
+To be able to easily share scripts with other repos and make sure that the
+exact dependencies needed to run the scripts are met we can use Nix.
+
+After changing submodules the lock file (`nix/dapp.nix`) needs to be updated
+using:
+
+```sh
+nix run -f https://github.com/icetan/dapp2nix/tarball/master -c dapp2nix > nix/dapp.nix
+```
+
+This is to avoid downloading all submodules when installing the deploy
```

Comment:
I don't understand this. Isn't dapp2nix's main goal reproducibility?

Reply:
Yes, and to avoid having to recursively fetch all submodules to get the complete dependency tree.

Reply:
OK, but it's unclear what "this" refers to here. Maybe move the sentence up, after the first paragraph?